packages/teams-js/src/internal/validDomains.ts

```diff
@@ -0,0 +1,104 @@
+import { GlobalVars } from '../internal/globalVars';
+import { validDomainsCdnEndpoint, validOriginsFallback } from './constants';
+import { getLogger } from './telemetry';
+import { isValidHttpsURL } from './utils';
+
+let validOrigins: string[] = [];
+export async function prefetchDomainsFromCDN(): Promise<void> {
```

Comment:
>prefetchDomainsFromCDN [](http://example.com/codeflow?start=22&length=22)

Rather than writing a unique fetch call here, could this function just call retrieveDomainsFromCDNAndStore and then ignore the return value? Is only fetching the value from the CDN actually helpful? Or would the best prefetching behavior to be to fetch and then store it?